src/app/components/program-details/program-details.css

```
@@ -468,3 +472,194 @@ p {
+        justify-content: center;
+        /* centers the single card */
+    }
+
+    .impact-header {
+        width: 100%;
+         gap: 2px;
+    }
+}
+
+
+::ng-deep .mat-mdc-dialog-container {
```

Stylelint (17.9.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)